packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx

```diff
@@ -33,18 +37,28 @@ export const BentoMenu = () => {
 
   const { env } = useConfig();
   const ftlMsgResolver = useFtlMsgResolver();
+  const experiments = useExperiments();
+  const nimbusUserId = experiments?.nimbusUserId;
+  const exampleFeature = experiments?.features?.['example-feature'];
+  const emojiExperiment = exampleFeature && exampleFeature?.['emoji'];
 
   useEffect(() => {
     if (isRevealed) {
-      GleanMetrics.accountPref.bentoView();
+      GleanMetrics.accountPref.bentoView({
+        event: { nimbusUserId },
+      });
```

Comment:
Related to the second additional notes comment, I assigned a test `nimbusUserId` within the `useExperiments` hook so that I could verify the extra was added:

<img width="353" alt="Screenshot 2025-03-17 at 1 20 18 PM" src="https://github.com/user-attachments/assets/cbf835a3-4d82-4fcb-8912-2bace13bfd5a" />